.github/workflows/podman-aws-e2e.yaml

```diff
@@ -10,7 +10,7 @@ permissions:
 
 jobs:
   rune2e:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
```suggestion
    # we are using ubuntu 24.04 because it come with version 4.9.3 of podman https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md
    runs-on: ubuntu-24.04
```

Reply:
Thanks, good idea! I wrote a slightly different comment in https://github.com/mesosphere/konvoy-image-builder/pull/1116/commits/9df4f6401dadfbb2d3b82db38cb293e0a2c06f40.